packages/dashboard/src/components/Header/Header.tsx

```diff
@@ -17,7 +17,7 @@ const Header = (props: HeaderProps) => {
   const tabs: Tab[] = ['Dashboard Description', 'Data Table Settings', 'Dashboard Preview']
   const { visualizationKey, subEditor } = props
   const { config, setParentConfig, tabSelected, data } = useContext(DashboardContext)
-  if (!config) return null
```

Comment:
It seems like this, or another check for config is needed, because we're accessing it below.